docs/index.md

```diff
@@ -9,16 +9,14 @@
 
 ROCm is an open-source software platform optimized to extract HPC and AI workload
 performance from AMD Instinct accelerators and AMD Radeon GPUs while maintaining
-compatibility with industry software frameworks. For more information, see [What is ROCm?](./what-is-rocm.rst)
+compatibility with industry software frameworks. For more information, see
+[What is ROCm?](./what-is-rocm.rst)
 
-If you're using AMD Radeon™ PRO or Radeon GPUs in a workstation setting with a display connected, review {doc}`Radeon-specific ROCm documentation<radeon:index>`.
-
-Installation instructions are available from:
+ROCm supports multiple programming languages and programming interfaces such as
+{doc}`HIP (Heterogeneous-Compute Interface for Portability)<hip:index>`, OpenCL,
+and OpenMP, as explained in the [Programming guide](./how-to/programming_guide.rst).
```

Comment:
```suggestion
and OpenMP, as explained in the [HIP Programming guide](./how-to/programming_guide.rst).
```

Reply:
The HIP programming guide section is at HIP side. That's why I would keep the original Programming guide